packages/vda-web3-client/src/config.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable prettier/prettier */
+import { Signer } from '@ethersproject/abstract-signer'
+
+import { BigNumber } from '@ethersproject/bignumber'
+import { Contract, ContractFactory } from '@ethersproject/contracts'
+import { JsonRpcProvider, Provider } from '@ethersproject/providers'
+import { ethers } from 'ethers'
+
+// linked package
+// import DidRegistryContract from 'vda-did-registry'
+import { DEFAULT_REGISTRY_ADDRESS, knownNetworks } from './helpers'
+
+const DidRegistryContract = require('./contract/VeridaDIDRegistry.json')
```

Comment:
Why is this here? This library is generic for to support any contract.

Reply:
Removed